### PR TITLE
Multiple fixes

### LINKED ISSUE #147: OS error: Permission denied at version `0.5`
`Spawning of process failed. (Error was: Additional info: "Could not find command: \'~/.choosenim/toolchains/nim-1.0.4/bin/nimble\'. OS error: Permission denied`

### LINKED ISSUE #151: CI does not name binaries correctly
The CI should name the binaries in the same way as is done for all previous releases.

CC @genotrance

### PR DESCRIPTION
**Pull request by genotrance**
Fix #147 - workaround rather since we don't have root cause yet
Fix #151 - correct binary names in CI
Add testament to list of proxies

### CODE REVIEW

File: src/choosenim/builder.nim [hunk 1]
```diff
@@ -73,6 +73,19 @@ proc buildTools() =
...
+  import posix
+
+  proc setPermissions() =
+    ## Assumes that CWD contains the compiler
+    let binDir = getCurrentDir() / "bin"
+    for kind, path in walkDir(binDir):
+      if kind == pcFile:
+        # Set to 755 = rwxrw-rw- in octal
+        # = 493 in decimal
+        discard chmod(path, 493)
```
Comment (dom96) [inline src/choosenim/builder.nim]: You shouldn't need posix for this, can you just use the `os` module?

Comment (genotrance) [reply src/choosenim/builder.nim]: Fixed.

File: src/choosenim/builder.nim [hunk 2]
```diff
@@ -73,6 +73,19 @@ proc buildTools() =
...
+    ## Assumes that CWD contains the compiler
+    let binDir = getCurrentDir() / "bin"
+    for kind, path in walkDir(binDir):
+      if kind == pcFile:
+        setFilePermissions(path,
+                           {fpUserRead, fpUserWrite, fpUserExec,
+                            fpGroupRead, fpGroupExec,
+                            fpOthersRead, fpOthersExec}
+        )
+
```
Comment (dom96) [inline src/choosenim/builder.nim]: Another thing that would be nice would be a `debug` output to say that these file permission changes are taking place. But I'll leave that to another PR if you agree that it's a good idea.

Comment (genotrance) [reply src/choosenim/builder.nim]: To do that, we will have to first check the extracted permissions. Do you want to add that additional code?

Comment (dom96) [reply src/choosenim/builder.nim]: Huh? Just show the message always, even if the permissions aren't being changed. It'll be hidden anyway most of the time.

Comment (genotrance) [reply src/choosenim/builder.nim]: Done in PR #158.

**Pull request merged**